### ISSUE TITLE
Sphinx not rendering documentation for .pyx files

### ISSUE THREAD

**Issue by alistairbntl** [erdc-cm/proteus#415]
Seeing how useful Cython, it think its worth finding a solution for this.

**Comment by adimako** [comment]
@alistairbntl, I noticed that for the WaveTools.pyx file

**Comment by alistairbntl** [comment]
Yeah, that was the one I was having trouble with.  I was playing around with it yesterday but haven't found a solution just yet.

**Comment by adimako** [comment]
@alistairbntl is this still a thing? For waveTools I moved towards  a module.py + module.pxd form (not .pyx) which allows to use documentation via the .py file. Is this relevant? https://stackoverflow.com/questions/10057476/how-do-i-use-sphinx-with-cython
